sql/moz-fx-data-shared-prod/jira_service_desk_derived/user_v1/query.py

```diff
@@ -0,0 +1,147 @@
+
+    def load_jira_user_data(self, destination_table: str, users: list[dict], truncate=True):
+        """Load downloaded data to BQ table.
+        https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-json#python
+        """
+        credentials, project = google.auth.default(
+            scopes=[
+                "https://www.googleapis.com/auth/cloud-platform",
+                "https://www.googleapis.com/auth/bigquery",
+            ]
+        )
+        client = bigquery.Client(credentials=credentials, project=project)
+        write_disposition = bigquery.WriteDisposition.WRITE_TRUNCATE if truncate else bigquery.WriteDisposition.WRITE_APPEND
+        
+        job_config = bigquery.LoadJobConfig(
+            schema=[
+                bigquery.SchemaField("account_id", "STRING"),
+                bigquery.SchemaField("account_status", "STRING"),                
+                bigquery.SchemaField("name", "STRING"),
+            ],
+            autodetect=False,
+            write_disposition=write_disposition,
+            source_format=bigquery.SourceFormat.NEWLINE_DELIMITED_JSON,
+        )
+
+        job = client.load_table_from_json(
+            users, destination_table, job_config=job_config
+        )
+        job.result()
+
+
+class JiraAPI:
+    def __init__(self, args) -> None:
+        self.logger = logging.getLogger(self.__class__.__name__)
+
+        self.secrets_dict = {
+          "jira_username": "$bqetl_jira_service_desk__jira_username",
+          "jira_token": "$bqetl_jira_service_desk__jira_token",
+        }
+        self.secrets_dict = {key:os.path.expandvars(self.secrets_dict[key]) for (key,value) in self.secrets_dict.items() }
+        self.base_jira_url = args.base_jira_url
+        self.auth = HTTPBasicAuth(self.secrets_dict.get('jira_username'), self.secrets_dict.get('jira_token'))
+
+    def get_users_paged(self, max_results=300):
+        startAt = 0
+        headers = {"Accept": "application/json"}
+
+        while True:           
+            url = (
+                self.base_jira_url
+                + f"/rest/api/3/users/search?query=+&maxResults={max_results}&startAt={startAt}"
+            )
+            try:
+                response = requests.request("GET", url, headers=headers, auth=self.auth)
+            except Exception as e:
+                self.logger.error(str(e))
+                self.logger.critical("Failed while getting Jira users")
+                sys.exit(1)
+            is_success = 299 >= response.status_code >= 200
+            if not is_success:
+                self.logger.error(f"ERROR: response.status_code = {response.status_code}")
+                self.logger.error(f"ERROR: response.text = {response.text}")
+                self.logger.error(f"ERROR: response.reason = {response.reason}")
+                self.logger.critical("Failed while getting Jira users")
+                sys.exit(1)
+                
+            users = json.loads(response.text)
```

Comment:
```suggestion
            users = response.json()
```

This should work too